ArcanicVosforator/src/main/java/org/example/arcanicvosforator/ArcanicVosforatorInterface.java

```diff
@@ -378,558 +223,426 @@ private void selectArcaneEvent() {
-                    getData().addToGrandTotalVosforCount();
+                    addToVosforCount(6, quantity);
                     break;
                 case "Cascadia Flare":
-                    getData().addToTotalVosforCount(getData().getVosforRates().get(6), quantity);
-                    getData().addToGrandTotalVosforCount();
+                    addToVosforCount(6, quantity);
                     break;
                 case "Eternal Onslaught":
-                    getData().addToTotalVosforCount(getData().getVosforRates().get(6), quantity);
-                    getData().addToGrandTotalVosforCount();
+                    addToVosforCount(6, quantity);
                     break;
                 case "Cascadia Empowered":
-                    getData().addToTotalVosforCount(getData().getVosforRates().get(6), quantity);
-                    getData().addToGrandTotalVosforCount();
+                    addToVosforCount(6, quantity);
                     break;
                 case "Emergence Renewed":
-                    getData().addToTotalVosforCount(getData().getVosforRates().get(6), quantity);
-                    getData().addToGrandTotalVosforCount();
+                    addToVosforCount(6, quantity);
                     break;
                 case "Molt Efficiency":
-                    getData().addToTotalVosforCount(getData().getVosforRates().get(6), quantity);
-                    getData().addToGrandTotalVosforCount();
+                    addToVosforCount(6, quantity);
                     break;
                 case "Eternal Logistics":
-                    getData().addToTotalVosforCount(getData().getVosforRates().get(6), quantity);
-                    getData().addToGrandTotalVosforCount();
+                    addToVosforCount(6, quantity);
                     break;
                 case "Molt Reconstruct":
-                    getData().addToTotalVosforCount(getData().getVosforRates().get(6), quantity);
-                    getData().addToGrandTotalVosforCount();
+                    addToVosforCount(6, quantity);
                     break;
                 case "Cascadia Overcharge":
-                    getData().addToTotalVosforCount(getData().getVosforRates().get(6), quantity);
-                    getData().addToGrandTotalVosforCount();
+                    addToVosforCount(6, quantity);
                     break;
                 case "Emergence Dissipate":
-                    getData().addToTotalVosforCount(getData().getVosforRates().get(6), quantity);
-                    getData().addToGrandTotalVosforCount();
+                    addToVosforCount(6, quantity);
                     break;
                 case "Molt Augmented":
-                    getData().addToTotalVosforCount(getData().getVosforRates().get(6), quantity);
-                    getData().addToGrandTotalVosforCount();
+                    addToVosforCount(6, quantity);
```

Comment:
small method to minimize the code reuse and improve readablity of the switch.
logic is truly unchaged, just minimizing reused code.

also makes it easier to expand in the future with more aracane additions.